README.rst

```diff
@@ -1,22 +1,69 @@
-FIWARE Policy Manager GE
-____________________
+================================
+FIWARE Policy Manager GE - Cloto
+================================
 
 
 | |Build Status| |Coverage Status| |Pypi Version| |Pypi License|
 
 
-Description
-===========
+This is the code repository for **FIWARE Policy Manager GE - Cloto**, a server that provides an API-REST to
```

Comment:
Before this content, you should add a index to the different sections, after it you have to add a Section called Introduction and start it with:
"This is the code repository for FIWARE Policy Manager GE - Cloto, a..."